schema/v1alpha1/managementcontroller.proto

```
@@ -0,0 +1,1055 @@
+// Management controller type enumeration
+enum ManagementControllerType {
+  MANAGEMENT_CONTROLLER_TYPE_UNSPECIFIED = 0;
+  MANAGEMENT_CONTROLLER_TYPE_BMC = 1;
+  MANAGEMENT_CONTROLLER_TYPE_SATELLITE = 2;
+  MANAGEMENT_CONTROLLER_TYPE_CHASSIS_CONTROLLER = 3;
+  MANAGEMENT_CONTROLLER_TYPE_NODE_CONTROLLER = 4;
+  MANAGEMENT_CONTROLLER_TYPE_RACK_CONTROLLER = 5;
+}
+
+// Management controller status enumeration
+enum ManagementControllerStatus {
```

Buf (1.55.1)

symbol "schema.v1alpha1.ManagementControllerStatus" already defined at schema/v1alpha1/state.proto:115:6

(COMPILE)